apps/omg_watcher_info/test/omg_watcher_info/db/repo_test.exs

```diff
@@ -25,6 +25,15 @@ defmodule OMG.WatcherInfo.DB.RepoTest do
   require Utxo
 
   describe "DB.Repo.insert_all_chunked/3" do
+    # The current number of columns on the transaction table allow up to 8190
+    # transactions to be inserted using `DB.Repo.insert_all/3` before chunking must
+    # be done to avoid hitting postgres limits.
+    @max_txns_before_chunking 8191
+
+    # The current number of columns on the transaction table require a chunk size of
+    # 10922 to avoid hitting postgres limits.
```

Comment:
Can you also describe how to derive the number 10922? So we know how to re-calculate.

Reply:
Ah, this number is wrong. I made a mistake. Actually `@max_txns_before_chunking  == @txn_chunk_size == 8191` which makes sense.

The number `10_922` came from a bad implementation of the `"chunk_size/1 calculates the correct chunk...` test. Notice `inserted_at` and `updated_at` were not being added to the transaction.
```
block = insert(:block)
transaction = new_transaction(block.blknum, 1)
assert DB.Repo.chunk_size(transaction) == @txn_chunk_size
```
I've fixed the test and removed the constant `@txn_chunk_size` completely. But still I think testing `OMG.WatcherInfo.DB.Repo.chunk_size/1` directly is important and want to keep it.

If you wish to derive the number just change and an `IO.puts()` or `IO.inspect() in the `chunk_size/1` function. Then run the test `test "DB.Repo.insert_all for transactions (via postgres INSERT)...` in `OMG.WatcherInfo.DB.RepoTest` and see what it prints out. To verify its correctness change the `chunk_size/1` to `div(@max_params_count, fields_count(entry)) + 1` and run `test "DB.Repo.insert_all for transactions (via postgres INSERT)...` again and it will fail.

Reply:
Umm I was just expecting just this formula but thanks

```
Formula for chunk size = floor(postgres_param_limit / schema_column_count)
Transaction columns = 8
Therefore, testing chunk size = floor(65535 / 8) = 8191
```